test/simulation.jl

```diff
@@ -509,19 +540,34 @@ end
     F == A ∧ (C ∧ B)
   end
 
-  sim = eval(gensim(wedges01))
+  let sim = eval(gensim(wedges01))
```

Comment:
It seems the only difference here is the value of  `can_prealloc`; loop here as well.